gpt_engineer/core/ai.py

```diff
@@ -177,54 +134,6 @@ def start(self, system: str, user: str, step_name: str) -> List[Message]:
         ]
         return self.next(messages, step_name=step_name)
 
-    def fsystem(self, msg: str) -> SystemMessage:
```

Comment:
these wrapper methods add little value. They are strangely named and scrolling past them constantly to read this class was frustrating